### PR TITLE
Revert to plain-old rubo.

### LINKED ISSUE #3373: remove standard gem (revert to plain rubocop)
remove the standard gem in favor of plain rubocop

per discussion at dev planning https://docs.google.com/document/d/1gzKgbqnlCc_GctBBpoYTvZl22mNZflsbqzyz4xXJlho/edit#heading=h.hmax9277viw

Consider what, besides the bare minimum, to put in `.rubocop.yml` as:
1. we may want to explicitly have some non-default settings for rules (e.g. line length)
2. if there is a lot of stuff in logs or tmp or vendor or whatever, running rubocop can be much slower since it looks at every directory.

With this in mind, I recommend doing at least one of the following:
1. see https://github.com/sul-dlss/DeveloperPlaybook/blob/main/style/infra_template.rubocop.yml
2. spelunk for the pull request that switched to standard and consider the old rubocop.yml file from that, or, at a minimum, 
3. look at another app with a UI, such as ETD or PreAssembly and take appropriate settings from that. (Note that Argo used standard for a while and just got rid of it with sul-dlss/argo/pull/4200 ... and required follow on sul-dlss/argo/issues/4214 to address these very concerns)

### PR DESCRIPTION
**Pull request by justinlittman**
closes #3373

# Why was this change made? 🤔
!standard

# How was this change tested? 🤨

⚡ ⚠ If this change involves consuming from or writing to another service (or shared file system), ***run [integration test create_object_h2_spec.rb](https://github.com/sul-dlss/infrastructure-integration-test)*** and/or test manually in [stage|qa] environment, in addition to specs. ⚡

# Does your change introduce accessibility violations? 🩺

⚡ ⚠ Please ensure this change does not introduce accessibility violations (at the WCAG A or AA conformance levels); if it does, include a rationale. See the [Infrastructure accessibility guide](https://github.com/sul-dlss/DeveloperPlaybook/blob/main/best-practices/infra-accessibility.md) for more detail. ⚡

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -1,26 +1,673 @@
...
-  - rubocop-capybara
-  - rubocop-factory_bot
   - rubocop-performance
   - rubocop-rails
   - rubocop-rspec
+  - rubocop-capybara
+  - rubocop-factory_bot
+
+# Don't even think about it.
+# inherit_from: .rubocop_todo.yml
```
Comment (ndushay) [inline .rubocop.yml:9]: can you say something about why?   This isn't terribly helpful.  Also ... does it need to be first in the file?

**Review by ndushay — COMMENTED** [review]
anything beyond double quotes to single quotes to call out?   Otherwise, this is way too big to review easily.

File: .rubocop.yml (hunk 1 above)
Comment (justinlittman) [reply .rubocop.yml]: Removed.

**Comment by justinlittman** [comment]
@ndushay Nothing too interesting. I reformatted some long lines and added various `rubocop:disable` instead of indiscriminate `.rubocop_todo`.

**Comment by ndushay** [comment]
> @ndushay Nothing too interesting. I reformatted some long lines and added various `rubocop:disable` instead of indiscriminate `.rubocop_todo`.

Did you know there's an option you can use so rubocop_todo shows the offending files?  `rubocop --auto-gen-config --auto-gen-only-exclude`

https://docs.rubocop.org/rubocop/configuration.html#automatically-generated-configuration

**Comment by justinlittman** [comment]
Yes, but once we have a `.rubocop_todo` available, we tend to get lazy. For new codebases, I'd suggest that `.rubocop_todo` is an anti-pattern.

**Comment by ndushay** [comment]
> Yes, but once we have a `.rubocop_todo` available, we tend to get lazy. For new codebases, I'd suggest that `.rubocop_todo` is an anti-pattern.

only when we agree with the rules.  :-P

**Review by ndushay — APPROVED** [review]

**Pull request merged**